src/crypto/CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ target_link_libraries(cncrypto
     epee
     ${Boost_SYSTEM_LIBRARY}
   PRIVATE
+    sodium
```

Comment:
This is required for successful build of master branch.
It's not required for is_hdd() implementation itself.